lib/Modules/S2DCharmUtils/S2DCharmUtils.psm1

```diff
@@ -17,6 +25,10 @@ function Remove-UnhealthyStoragePools {
     #     formed a storage pool before, this one is listed with "Unknown" status and
     #     read-only mode. After disabling read-only flag, the storage pool becomes
     #     "Unhealthy" and it's unusable.
+    $winPSModules = Join-Path $env:windir "system32\windowspowershell\v1.0\Modules"
```

Comment:
With my version of juju `2.0-beta7`, on Nano, the `PATH` for jujud user is broken and does not contain these. This is a Juju problem.